firmware/lib/Gantry/Axis.cpp

```diff
@@ -204,18 +226,22 @@ void isr_limit_switch_y()
     uint32_t interrupt_time = millis();
     // If interrupts come faster than 10ms, assume it's a bounce and ignore
     if (interrupt_time - last_interrupt_time > 50) {
-        NVIC_DisableIRQ(axis_y.irq_velocity);
-        NVIC_DisableIRQ(axis_y.irq_accel);
+        stop_axis(&axis_x);
         DEBUG_PRINT_VAL("Y Home Limit switch has been hit", 1);
 
```

Comment:
nitpicky - preference for the word "reached" over "hit"

Reply:
I'm gonna go through and refactor all of the motor code print statements, so not gonna bother fixing this here